governance/pyth_staking_sdk/eslint.config.js

```diff
@@ -6,6 +6,7 @@ export default [
     rules: {
       "n/no-unpublished-import": "off",
       "unicorn/no-null": "off",
+      "unicorn/prefer-node-protocol": "off",
```

Comment:
I don't understand this

Reply:
it asking to import `node:crypto` instead of `crypto` which doesn't work for our setup.